clients/go/Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: all
+all : install-protoc generate-proto
+
+fmt:
+	@go mod tidy
+	@go fmt ./...
+
+staticcheck:
+	go run honnef.co/go/tools/cmd/staticcheck ./...
+
+install-protoc:
+	@echo "> installing dependencies..."
+	go get -u github.com/golang/protobuf/proto@v1.4.3
+	go get -u github.com/golang/protobuf/protoc-gen-go@v1.4.3
+	go get -u google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
+
+PROTO_PACKAGE=/proto
+generate-proto:
```

Comment:
@AkbaraliShaikh proto compilation process can be simplified. check for ref. 
https://github.com/odpf/guardian/blob/1f8270b354a9715e3cc29c7426ff5d126aea053a/Makefile#L61

Reply:
Planning to use the pre-compiled for now, so this change is not needed, I will update the PR.
go.buf.build/odpf/gw/odpf/proton

Reply:
Updated